index.d.ts

```diff
@@ -10,6 +10,13 @@ declare module '@ekreative/react-native-braintree' {
     currencyCode: string;
   }
 
+  export interface PayPalOptions {
```

Comment:
Could you create the `BaseOptions` interface?

[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)